packages/react-aria-components/test/Table.test.js

```diff
@@ -1874,7 +1874,7 @@ describe('Table', () => {
       expect(loaderRow).toHaveTextContent('spinner');
 
       let sentinel = tree.getByTestId('loadMoreSentinel');
-      expect(sentinel.parentElement).toHaveAttribute('inert');
+      expect(sentinel.parentElement.parentElement).toHaveAttribute('inert');
```

Comment:
seems brittle, can we just use sentinel.closest('[inert]') is truthy?

Reply:
sure, was mainly concerned about that possibly matching a higher level inert element and still passing, but I'll see about limiting it to the body